app/controllers/dams_resource_controller.rb

```diff
@@ -30,6 +30,17 @@ def show
     # get metadata from solr
     @document = get_single_doc_via_search(1, {:q => "id:#{params[:id]}"} )
 
+    # generate facet collection list for collection page only
+    if !@document.nil?
```

Comment:
Interesting, I'm trying to recall if there's a reason we would have had that test assert a 403 instead of a 404. What do you think would be technically correct despite the existing test? It seems like 404 (not found) is what is actually happening here.